assets/js/modules/forms.js

```diff
@@ -81,6 +82,45 @@ function handleExpandingDetails() {
     });
 }
 
+function handleConditionalRadios() {
+    forEach(document.querySelectorAll('.js-conditional-radios'), el => {
+        function setAttributes(radioEl) {
+            var inputIsChecked = radioEl.checked;
+            var conditionalEl = el.querySelector(`#${radioEl.getAttribute('aria-controls')}`);
+            if (conditionalEl) {
+                conditionalEl.setAttribute('aria-expanded', inputIsChecked);
```

Comment:
Not 100% sure that combining `aria-expanded` with `hidden` is the right semantics, but something to test.